src/reducers/auth.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import type {Action} from '../actions/types';
+
+type AuthState = {
+  state: 'mystery' | 'logged in' | 'logged out' | 'login failed',
+  user?: {}, // TODO(marcos): fill this in once you know the shape,
```

Comment:
this is a sort of cheat, flow lets you use an empty object as a type to say "it's object-like" and flow will only really check that you don't feed it an Array or somesuch, but it won't do any checking on the properties of the object, so this is a somewhat useful type, but it's mostly useless until you know which keys from `user` you depend on.